resources/outputs.js

```diff
@@ -145,7 +145,7 @@ const Outputs = new ThrowOnInvalidProxy({
     en: 'Stack',
     de: 'Sammeln',
     fr: 'Packez-vous',
-    ja: '頭割り',
+    ja: 'スタック',
     cn: '分摊',
```

Comment:
This is stack for other things instead of damage share, how can we call them?

RE: Should we be consistent in JA to call all the stack gimmick as スタック or something?

Reply:
頭割り is popular in Japanese.

Reply:
You might misunderstand me.
https://github.com/quisquous/cactbot/blob/126e5cf537d2d8c97ad26afaa9ee3bd84419a12e/resources/outputs.js#L134-L142
Here is the damage share, we call it 頭割り.

https://github.com/quisquous/cactbot/blob/126e5cf537d2d8c97ad26afaa9ee3bd84419a12e/resources/outputs.js#L143-L151
And here is get together, not damage share I think. What can we call it?

Reply:
Umm, maybe it should follow en so we call スタック ...?

Reply:
That's the point that I am not sure.
In my opinion, スタック is usually used in programing field as the data structure `stack`, not for refering "getting together".
What about following cn, that they always call it as 集合? or 集まる if prefer 和語?

Reply:
集合 is good idea!

Reply:
Ok, I'll change it tomorrow.
good night!